### DISCUSSION TITLE
Panel collapsible ability issue

### DISCUSSION THREAD

**Discussion by scramatte** [laravel/nova-issues#5877] [Q&A]
Hello,

It looks, since v4.2.6, you have added the ability for panels to be collapsible. Unfortunately, I don't found any reference in the documentation. Somebody has some clue about ?  It looks that exists methods "collapsedByDefault" and "collapsable"  but I don't see any change on the UI.

I've got a long detail page and would like to group and collapse some extra fields by default

Regards

**Comment by scramatte** [comment]
I've just discover that the feature works only on detail view. On Forms, panel are not collapsible. It's normal or it's bug ?  It should works on both views ?

**Reply by crynobone** [reply]
No, the feature is intended only for detail views

**Reply by scramatte** [reply]
Hi,  I've made a trick to get it working with detail view too.
It would be nice to add it to the next nova release.
Thank you

**Reply by dmason30** [reply]
I am against allowing them to be collapsable on from as collapsed panels would end up hiding validation errors.
